include/storage/io.hpp

```diff
@@ -15,17 +20,38 @@ namespace storage
 namespace io
 {
 
+// Reads the count of elements that is written in the file header and returns the number
+inline std::uint64_t readElementCount(boost::filesystem::ifstream &input_stream)
+{
+    std::uint64_t number_of_elements = 0;
+    input_stream.read((char *)&number_of_elements, sizeof(std::uint64_t));
+    return number_of_elements;
+}
+
+// To make function calls consistent, this function returns the fixed number of properties
+inline std::size_t readPropertiesCount() { return 1; }
+
+// Returns the number of bytes in a file
+inline std::size_t readNumberOfBytes(boost::filesystem::ifstream &timestamp_input_stream)
```

Comment:
This only works for timestamp streams? Then probably should have "timestamp" in the function name.